pom.xml

```
@@ -5,17 +5,16 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.10</version> <!-- CAUTION: do not go past 2.16 without doing something like https://github.com/jenkinsci/workflow-cps-plugin/pull/87 -->
+    <version>3.17</version> <!-- CAUTION: do not go past 2.16 without doing something like https://github.com/jenkinsci/workflow-cps-plugin/pull/87 -->
```

BTW I think the caution here is actually important, you need to update `scripts` in package.json like in https://github.com/jenkinsci/workflow-cps-plugin/pull/87 or the build process will be broken (silently?).

---

I think this also might be why the CI build is failing.

---

haha I was not sure as build works fine locally :) I will look at it.

---

I think you can go ahead and delete the TODO comment now:

```suggestion
    <version>3.17</version>
```